### PR TITLE
Remote news story CSS

### PR DESCRIPTION
**Pull request by drewvolz**
* Convert `css` located in `data` into `json` in `docs`
* Use local stylesheet if in development
* Use remote stylesheet if in production
* Rule for `figure` width
* Rule for `word-wrap` on long strings

### CODE REVIEW

**Comment by drewvolz** [comment]
Question: is it worth pulling in the stylesheet on each story, or should the content be fetched once in the news list and passed as a prop to the story view?

... so load it for each view, or once on fetch?

**Comment by hawkrives** [comment]
Once per view. Just insert the `<link>` element and let the WebView cache if it wants.

**Review by hawkrives — CHANGES_REQUESTED** [review]
This should be a `.css` file within `data/`, and be transformed to a `json` file in `docs/`. You should never commit files to `docs/`.

File: source/views/news/news-item.js [hunk 1]
```diff
@@ -28,43 +28,18 @@ export default function NewsItem(props: {
 }) {
   const {story, embedFeaturedImage} = props.navigation.state.params
 
+  // Use local or remote styles
+  let newsCSS = ''
+  if (process.env.NODE_ENV === 'development') {
+    let css = require('../../../docs/news-styles.json')
+    newsCSS = `<Style>${css}</Style>`
```
Comment (hawkrives) [inline source/views/news/news-item.js]: Lower-case `Style`. It's an html element.

Comment (drewvolz) [reply source/views/news/news-item.js]: oops

**Comment by drewvolz** [comment]
> Once per view. Just insert the <link> element and let the WebView cache if it wants.

So the way it currently works is okay

**Comment by drewvolz** [comment]
> This should be a .css file within `data/`, and be transformed to a `json` file in `docs/`

~Do you have any recommendations as to how we'd convert the `css` to `json`?~

I've found a way. I'll push shortly.

**Comment by drewvolz** [comment]
Fixed!

**Review by hawkrives — APPROVED** [review]
I think this looks good to me!

**Pull request merged**